src/components/AllComments.jsx

```diff
@@ -13,8 +13,40 @@ export const AllComments = () => {
         })
     }, [review_id]);
 
+    const [newComment, setNewComment] = useState("");
+    const [err, setErr] = useState("");
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const addComment = {
+            username: username.username,
+            body: newComment
+        };
+        api.postCommentToReview(review_id, addComment).then((addedComment) => {
+            setComments((currentComments) => {
+                return [addedComment, ...currentComments];
```

Comment:
This isn't optimisticly rendered since it happens in the then block after the api request.
Please change this to happen optimistically.